ast/ast.go

```diff
@@ -693,3 +728,10 @@ func (this ID) Walk(v Visitor) {
 	}
 	v.Visit(this)
 }
+
+func genIndent(lvl int) (idt string) {
```

Comment:
I can't find where this is used?

Reply:
Good catch, thanks. I was previously passing down the indentation level as int and generating the prefix each time before I decided to switch to the the indentation level being represented as the prefix itself. Removed it in the most recent commit.